media/css/m24/components/navigation-refresh.scss

```
@@ -318,21 +324,48 @@ $margin-top: 58px; // top margin offset for mobile navigation menu
 }
 
 .m24-c-menu-category-link {
-    display: flex;
-    font-size: $text-body-md;
-    justify-content: flex-end;
-    margin: $spacer-lg $spacer-lg 0;
-    padding: 0;
-
     a:link,
     a:visited {
         display: flex;
+        justify-content: space-between;
+        margin: $spacer-md 0 -1em;
+        padding: $spacer-sm;
+        background: var(--m24-light-orange);
```

```suggestion
        background: $m24-color-light-orange;
```

Let's use the tokens instead of CSS variables for now. I know we're planning to go all in on CSS vars but I don't want a mix of them to manage until we make the switch

---

This definitely was not a design choice, most likely just a WIP — or I wasn't able to locate the right token from the right lib that would be loaded at that point;) Thanks for getting the correct ones for me.